test/evergreen.yml

```diff
@@ -296,12 +299,15 @@ functions:
           powershell.exe  -NonInteractive '.\test\evergreen\build_windows.ps1' -configure 1 $DEFINED_EVERGREEN_CONFIG_FLAGS ${windows_configure_flags|}
         else
           echo "Using config flags $DEFINED_EVERGREEN_CONFIG_FLAGS ${posix_configure_flags|}"
-          # Fetch the gperftools library if needed. This will also get tcmalloc.
-          if [[ ${ENABLE_TCMALLOC|0} -eq 1 ]] && [ ! -d "./automation-scripts" ]; then
-            is_cmake_build=true
-            git clone git@github.com:wiredtiger/automation-scripts.git
-            . automation-scripts/evergreen/find_gperftools.sh ${s3_access_key} ${s3_secret_key} ${build_variant} $is_cmake_build
+
+          if [[ ${ENABLE_TCMALLOC|0} -eq 1 ]]; then
+            bin/bash test/evergreen/tcmalloc_install_or_build.sh ${build_variant}
```

Comment:
Nit: The above line #218 has set the shell as bash. Consider using `source` or `.` instead (prefer the latter to be consistent with the below line #313).

Reply:
If we use `source` or `.` we inherit shell options and would have to explicitly turn off `-e` option to prevent the build failing if we are unable to download pre-built binaries for the variant. I was using `./` but `/bin/bash` is more explicit and does not require the executable bits to be set.

Reply:
`source` or `.` is for executing a script in the current environment. It will leak it's internals to the current script.
`bash` is for executing the script as an external command.

A correct invocation would be either `bash` or `/bin/bash`.

Reply:
I see. A comment would be helpful for our future selves.